manage.yml

```diff
@@ -3,6 +3,7 @@ help_text: This is the {project_name} interactive shell
 
 # interactive shell
 shell:
+  ipython_auto_reload: true
```

Comment:
So if you make changes to the code, ipython will pick them up automatically and reload the modules? Sweet!!!